integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwSample.java

```diff
@@ -0,0 +1,473 @@
+      replaceStringInFile(Paths.get(sampleBase.toString(), "create-domain-inputs.yaml").toString(),
+              "#imagePullSecretName:", "imagePullSecretName: " + BASE_IMAGES_REPO_SECRET);
+      replaceStringInFile(Paths.get(sampleBase.toString(), "create-domain-inputs.yaml").toString(),
+              "rcuDatabaseURL: database:1521/service", "rcuDatabaseURL: " + dbUrl);
+    });
+  }
+
+  /**
+   * Start Oracle DB instance, create rcu pod and load database schema in the specified namespace.
+   *
+   * @param dbImage image name of database
+   * @param fmwImage image name of FMW
+   * @param rcuSchemaPrefix rcu SchemaPrefix
+   * @param dbNamespace namespace where DB and RCU schema are going to start
+   * @param dbPort NodePort of DB
+   * @param dbUrl URL of DB
+   * @throws Exception if any error occurs when setting up RCU database
+   */
+
+  private static void setupDBandRCUschemaBySample(String dbImage, String fmwImage,
+       String rcuSchemaPrefix, String dbNamespace, int dbPort, String dbUrl) throws ApiException {
+    LoggingFacade logger = getLogger();
+
+    setupSample();
+    // create pull secrets when running in non Kind Kubernetes cluster
+    // this secret is used only for non-kind cluster
+    createSecretForBaseImages(dbNamespace);
+
+    logger.info("Start Oracle DB with dbImage: {0}, dbPort: {1}, dbNamespace: {2}",
+        dbImage, dbPort, dbNamespace);
+    startOracleDB(dbImage, dbPort, dbNamespace);
+    logger.info("Create RCU schema with fmwImage: {0}, rcuSchemaPrefix: {1}, dbUrl: {2}, "
+        + " dbNamespace: {3}:", fmwImage, rcuSchemaPrefix, dbUrl, dbNamespace);
+    createRcuSchema(fmwImage, rcuSchemaPrefix, dbUrl, dbNamespace);
+
+  }
+
+  /**
+   * Start Oracle DB pod and service in the specified namespace.
+   *
+   * @param dbBaseImageName full image name for DB deployment
+   * @param dbPort NodePort of DB
+   * @param dbNamespace namespace where DB instance is going to start
+   */
+  private static synchronized void startOracleDB(String dbBaseImageName, int dbPort, String dbNamespace) {
+    LoggingFacade logger = getLogger();
+
+    Path dbSamplePathBase = Paths.get(tempSamplePath.toString(), "/scripts/create-oracle-db-service/");
+    String script = Paths.get(dbSamplePathBase.toString(),  "start-db-service.sh").toString();
+    logger.info("Script for startOracleDB: {0}", script);
+
+    String command = script
+        + " -i " + dbBaseImageName
+        + " -p " + dbPort
+        + " -s " + BASE_IMAGES_REPO_SECRET
+        + " -n " + dbNamespace;
+    logger.info("Command for startOracleDb: {0}", command);
+    assertTrue(() -> Command.withParams(
+        defaultCommandParams()
+            .command(command)
+            .saveResults(true)
+            .redirect(true))
+        .execute(), "Failed to execute command: " + command);
+
+    // sleep for a while to make sure the DB pod is created
+    try {
+      Thread.sleep(2 * 1000);
```

Comment:
is 2 seconds enough? do you want to replace it with a dynamic wait?

Reply:
I increased it to 10 seconds.
I tried dynamic one before without success.